addon/services/airbrake.js

```diff
@@ -3,7 +3,8 @@ import getClient from '../utils/get-client';
 import setEnvironment from '../filters/environment';
 import loggerReporter from '../reporters/logger';
 import setSession from '../filters/session';
-import getOwner from 'ember-getowner-polyfill';
```

Comment:
Ah no, I am full of it :-) fat fingered a change to package.json. Nothing to see here!